utils/makeip/gimp/file-mr.py

```diff
@@ -14,13 +14,13 @@
 from gimpfu import *
 from array import array
 
-def to_bytes(n, length, endianess='big'):
```

Comment:
Not too familiar with python syntax. I think this is a variable name, and it looks like it's being fixed throughout, but please double check to make sure this doesn't break the script.